servicebroker/lib/api/api.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var settings = JSON.parse(
```

Comment:
it is used to define a port when start service broker app. It is useless when cf app, but it should be useful if we want to define another port with bosh deploy. Is that right?